src/cc/frontends/clang/kbuild_helper.cc

```diff
@@ -74,28 +74,29 @@ int KBuildHelper::get_flags(const char *uname_machine, vector<string> *cflags) {
   cflags->push_back("-isystem");
   cflags->push_back("/virtual/lib/clang/include");
 
-  // The include order from kernel top Makefile:
-  //
-  // # Use USERINCLUDE when you must reference the UAPI directories only.
-  // USERINCLUDE    := \
-  //                 -I$(srctree)/arch/$(SRCARCH)/include/uapi \
-  //                 -I$(objtree)/arch/$(SRCARCH)/include/generated/uapi \
-  //                 -I$(srctree)/include/uapi \
-  //                 -I$(objtree)/include/generated/uapi \
-  //                 -include $(srctree)/include/linux/kconfig.h
-  //
-  // # Use LINUXINCLUDE when you must reference the include/ directory.
-  // # Needed to be compatible with the O= option
-  // LINUXINCLUDE    := \
-  //                 -I$(srctree)/arch/$(SRCARCH)/include \
-  //                 -I$(objtree)/arch/$(SRCARCH)/include/generated \
-  //                 $(if $(building_out_of_srctree),-I$(srctree)/include) \
-  //                 -I$(objtree)/include \
-  //                 $(USERINCLUDE)
-  //
-  // Some distros such as openSUSE/SUSE and Debian splits the headers between
-  // source/ and build/. In this case, just $(srctree) is source/ and
-  // $(objtree) is build/.
+  /* The include order from kernel top Makefile:
+   *
+   * # Use USERINCLUDE when you must reference the UAPI directories only.
+   * USERINCLUDE    := \
+   *                 -I$(srctree)/arch/$(SRCARCH)/include/uapi \
+   *                 -I$(objtree)/arch/$(SRCARCH)/include/generated/uapi \
+   *                 -I$(srctree)/include/uapi \
+   *                 -I$(objtree)/include/generated/uapi \
+   *                 -include $(srctree)/include/linux/kconfig.h
+   *
+   * # Use LINUXINCLUDE when you must reference the include/ directory.
+   * # Needed to be compatible with the O= option
+   * LINUXINCLUDE    := \
+   *                 -I$(srctree)/arch/$(SRCARCH)/include \
+   *                 -I$(objtree)/arch/$(SRCARCH)/include/generated \
+   *                 $(if $(building_out_of_srctree),-I$(srctree)/include) \
+   *                 -I$(objtree)/include \
+   *                 $(USERINCLUDE)
+   *
+   * Some distros such as openSUSE/SUSE and Debian splits the headers between
+   * source/ and build/. In this case, just $(srctree) is source/ and
+   * $(objtree) is build/.
+   */
```

Comment:
I think existing comment is okay, there is no need to change. This is C++ any way.